### DISCUSSION TITLE
Dubious sponsors of the Mastodon project on joinmastodon.org

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#18759] [General]
When visiting the joinmastodon.org project site, I noticed at least ten sposors that I would personally classify as very questionable. 

These would be at least: 

- Row Two Number Two
- Row Two Number Six
- Row three number one and two
- Row four number one, two and four
- Row five number one to three 

I have marked them in the attached screenshot.

I am a great friend of the Mastodon project and the Fediverse, but it seems to me that with such dubious sponsors, mastodon could suffer a similar fate to PeerTube, which unfortunately has a very tarnished image due to the fact that many right-wing extremist parties or organisations run instances and provide content on it. 

Therefore my question to @Gargron  (and @ClearlyClaire ) would be: Is there currently a guideline regarding the sponsors of the Mastodon project and if not, would it make sense to introduce such a guideline in view of the dubious sponsors mentioned and to revise sponsors who are contrary to this guideline and thus the values of the project?

![mastodon-project-sponsoren-mhm](https://user-images.githubusercontent.com/47597136/177053836-b63fe59d-2107-4e94-a9fb-43ac401daedd.png)

**Comment by FediVideos** [comment]
Yeah, have to agree here, does Mastodon really need money from casinos? It does sort of give a cheap feel to the whole project.

**Comment by Gargron** [comment]
> does Mastodon really need money

The answer to that question is yes. There are more people working on Mastodon right now in an official capacity than ever before, and it does not come cheap. I'm sorry you do not like some of our sponsors, but anyone can sponsor the project through Patreon and our custom portal, and these are the companies and individuals that want to do it. In the end, a sponsorship is just a donation with a link as a reward and no other strings attached, and I think many would find that more ethical than e.g. Firefox being supported by Google in exchange for making Google the default search engine (i.e. influencing functionality).
